src/onboarding/registration/NameAndPicture.test.tsx

```diff
@@ -21,6 +24,19 @@ jest.mock('statsig-react-native', () => ({
   },
 }))
 
+const mockRng = jest.fn()
+jest.mock('seedrandom', () => ({
```

Comment:
This is fine, but given the same random seed, a random number generator should give a deterministic result. That's why I recommended adding the seed parameter-- so you wouldn't have to mock the rng

Reply:
that being said, it does seem handy that this lets you choose which word the function will select, so maybe it's still worth having it mocked out

Reply:
I think I'll keep the tests as is and remove the seed value param in `_chooseRandomWord`

Reply:
Also removing the package I installed for Math.random instead.